src/processes/filters.rs

```diff
@@ -191,6 +205,10 @@ pub mod tests {
         };
         assert!(filter.accept(&process, None));
 
+        // tests that fuzzy search works
+        process.cmd_path = Some("/taest".to_string());
```

Comment:
I may be wrong with this but shouldn't we use 'tst'?
I have tested it with skim and 'taest' didn't find test.txt file, 'tst' did

Reply:
Apparently no. For me, it works conversely. I've checked the source code of skim in order to make sure that I'm passing parameters properly to the `fuzzy_match` function. This is strange :)

Reply:
To summarize how this code works for me:
- query: tst, cmd_path: test => match
- query: test, cmd_path: tst => no match
- query: st, cmd_path: test => match
- query: taest, cmd_path: test => no match

I think these are enough examples to show how fuzzy matching works.

Reply:
Yep, this is exactly how I imagine this works. It's simply contains on steroids.

Reply:
Exactly! So, does it mean you're happy with it? If yes, please address my comment in the thread above: https://github.com/jacek-kurlit/pik/pull/76#discussion_r1776021350

Reply:
I'm happy with this approach, closing